commit dbc7dffc7b9ca63e72ebe51c9137f286bfe77c7e
Author: Alejandro Alvarez Ayllon <a.alvarezayllon@gmail.com>
Date:   Thu Dec 8 21:33:23 2022 +0100

    Build shared libraries, fix install

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f914e78b..3167e6b8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -363,7 +363,7 @@ list(REMOVE_ITEM __tmp_srcs "${ONNX_ROOT}/onnx/cpp2py_export.cc")
 list(REMOVE_ITEM __tmp_srcs ${onnx_gtests_src})
 list(APPEND ONNX_SRCS ${__tmp_srcs})
 
-add_library(onnx_proto ${ONNX_PROTO_SRCS} ${ONNX_PROTO_HDRS})
+add_library(onnx_proto SHARED ${ONNX_PROTO_SRCS} ${ONNX_PROTO_HDRS})
 add_dependencies(onnx_proto gen_onnx_operators_proto gen_onnx_data_proto)
 target_include_directories(onnx_proto PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
@@ -419,7 +419,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "AIX")
   # So, create a object library
   add_library(onnx OBJECT ${ONNX_SRCS})
 else()
-  add_library(onnx ${ONNX_SRCS})
+  add_library(onnx SHARED ${ONNX_SRCS})
 endif()
 
 target_include_directories(onnx PUBLIC
@@ -675,6 +675,12 @@ install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/onnx
         FILES_MATCHING
         PATTERN "*.h")
 
+install(DIRECTORY ${ONNX_ROOT}/onnx
+        DESTINATION "${PY_SITEARCH}"
+        FILES_MATCHING
+        PATTERN "*.py"
+        PATTERN "test/*" EXCLUDE )
+
 configure_file(
   ${PROJECT_SOURCE_DIR}/cmake/ONNXConfigVersion.cmake.in
   ${PROJECT_BINARY_DIR}/ONNXConfigVersion.cmake
@@ -693,10 +699,13 @@ install(TARGETS
   onnx onnx_proto
   onnxifi onnxifi_dummy onnxifi_loader
   EXPORT ONNXTargets DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(TARGETS
+  onnx_cpp2py_export
+  DESTINATION "${PY_SITEARCH}/onnx")
 
 if(NOT ANDROID AND NOT IOS)
   install(TARGETS onnxifi_wrapper
-    EXPORT ONNXTargets DESTINATION lib)
+    EXPORT ONNXTargets DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
 
 if(ONNXIFI_DUMMY_BACKEND)
