commit 62e47a470ea4bed92077fff03b731e99989707be
Author: Alejandro Alvarez Ayllon <a.alvarezayllon@gmail.com>
Date:   Thu Dec 8 22:26:12 2022 +0100

    Fix setup.cfg to run during packaging

diff --git a/setup.cfg b/setup.cfg
index b8983afe..a2db6305 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,11 +1,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
+[tox:tox]
+envlist=py311
+
+[testenv]
+deps=pytest
+commands=pytest
+
 [aliases]
 test=pytest
 
 [tool:pytest]
-addopts = --nbval --current-env
-testpaths = onnx/test onnx/examples
+addopts = --ignore=onnx/test/test_backend_test.py --ignore=onnx/test/reference_evaluator_backend_test.py
+testpaths = onnx/test
 
 [metadata]
 license_files = LICENSE
