diff --git a/setup.cfg b/setup.cfg
index 219f65b8..aa09ce88 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,11 +1,18 @@
 # SPDX-License-Identifier: Apache-2.0
 
+[tox:tox]
+envlist=py311
+
+[testenv]
+deps=pytest
+commands=pytest
+
 [aliases]
 test=pytest
 
 [tool:pytest]
-addopts = --nbval --current-env
-testpaths = onnx/test onnx/examples
+addopts = --ignore=onnx/test/test_backend_test.py
+testpaths = onnx/test
 
 [metadata]
 license_files = LICENSE
@@ -50,7 +57,7 @@ warn_no_return = True
 warn_redundant_casts = True
 warn_incomplete_stub = True
 # TODO disallow_untyped_calls = True
-check_untyped_defs = True
+eck_untyped_defs = True
 disallow_any_generics = True
 no_implicit_optional = True
 # TODO disallow_incomplete_defs = True
